content/numerical/Simplex.h

```diff
@@ -29,22 +29,22 @@ struct LPSolver {
 	vvd D;
 
 	LPSolver(const vvd& A, const vd& b, const vd& c) :
-		m(sz(b)), n(sz(c)), N(n+1), B(m), D(m+2, vd(n+2)) {
-			rep(i,0,m) rep(j,0,n) D[i][j] = A[i][j];
-			rep(i,0,m) { B[i] = n+i; D[i][n] = -1; D[i][n+1] = b[i];}
-			rep(j,0,n) { N[j] = j; D[m][j] = -c[j]; }
+		m(SZ(b)), n(SZ(c)), N(n+1), B(m), D(m+2, vd(n+2)) {
+			fore(i,0,m) fore(j,0,n) D[i][j] = A[i][j];
+			fore(i,0,m) {B[i]=n+i; D[i][n]=-1; D[i][n+1]=b[i];}
+			fore(j,0,n) { N[j] = j; D[m][j] = -c[j]; }
```

Comment:
```suggestion
			m(SZ(b)), n(SZ(c)), N(n+1), B(m), D(m+2, vd(n+2)) {
		fore(i,0,m) fore(j,0,n) D[i][j] = A[i][j];
		fore(i,0,m) {B[i]=n+i; D[i][n]=-1; D[i][n+1]=b[i];}
		fore(j,0,n) { N[j] = j; D[m][j] = -c[j]; }
```
Estaba como está de antes, pero creo que queda mejor así